getl/fileregistry/delta_diff.py

```diff
@@ -0,0 +1,61 @@
+from datetime import datetime
+
+from pyspark.sql import DataFrame, functions as F, types as T
+
+from ..block import BlockConfig
+from ..fileregistry import fileregistry_utils as fr_utils
+
+
+class DeltaDiff:
+    schema = T.StructType([T.StructField("version_date", T.TimestampType(), False)])
+
+    def __init__(self, bconf: BlockConfig) -> None:
+        self.file_registry_path = bconf.get("BasePath")
```

Comment:
Could we deprecate BasePath in favour of FileRegistryPath or just Path? Both would work but BasePath gives a warning.